packages/opam-dune-lint/opam-dune-lint.0.3/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Ensure dune and opam dependencies are consistent"
+description:
+  "opam-dune-lint checks that all ocamlfind libraries listed as dune dependencies have corresponding opam dependencies listed in the opam files. If not, it offers to add them (either to your opam files, or to your dune-project if you're generating your opam files from that)."
+maintainer: ["alpha@tarides.com" "Tim McGilchrist <timmcgil@gmail.com>"]
```

Comment:
It seems that the list of maintainers is modified. Just want to point this out and make sure it is intentional.

```diff
< maintainer: ["talex5@gmail.com"]
---
> maintainer: ["alpha@tarides.com" "Tim McGilchrist <timmcgil@gmail.com>"]
```

Reply:
Yes it is.

cc: @tmcgilchrist